djredcap/management/subcommands/inspect.py

```diff
@@ -59,8 +59,24 @@ def handle(self, fileName=None, *args, **options):
         dialect = csv.Sniffer().sniff(fin.read(1024))
         fin.seek(0)
         reader = csv.DictReader(fin, fieldnames=header_keys, dialect=dialect)
+	
+        
+        # Open the file in a normal reader to see if the file is a REDCap data dictionary
+        # A REDCap data Dictionary begins with the four necessary columns:
+        # Variable/Field Name | Form Name | Field Type | Field Label in any order
+        header = reader.next()
 
-        reader.next()
-        if fileName.find('.json') == -1:
+        if header['field_name'] != 'Variable / Field Name' or \
+            header['form_name'] != 'Form Name' or \
+            header['field_type'] != 'Field Type' or \
+            header['field_label'] != 'Field Label':            
+
+            sys.stderr.write(fileName + "\n")

```

Comment:
This is a nit, but use single quotes for `'\n'` to be consistent with the surrounding code.